README.md

```diff
@@ -56,15 +55,20 @@ to combine them with existing itp-files see the wiki pages.
 ### Initial structure generation
 To generate an initial structure run:
 ```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule>
+polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -dens <density>
+```
+or:
+```
+polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -box <x, y, z>
 ```
 In order to append coordinates to an already existing coordinate file run:
 ```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -c <init_coords.gro>
+polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -c <init_coords.gro> -box/-dens
```

Comment:
I would just give e.g. `-box 10 10 10`, or `-dens 1` so that the command will actually run. Up to you though